app/assets/javascripts/ideas.js

```diff
@@ -1,13 +1,14 @@
 $(document).ready(function(){
-  window.onload = fetchIdeas
+  var $body = $('body');
```

Comment:
This is a great refactor! I like how you created a variable instead of calling $('body') each time!